src/connector/src/sink/encoder/mod.rs

```diff
@@ -134,8 +140,8 @@ pub enum CustomJsonType {
     // The internal order of the struct should follow the insertion order.
     // The decimal needs verification and calibration.
     Doris(HashMap<String, (u8, u8)>),
-    // Bigquery's json need date is string.
-    Bigquery,
+    // Es's json need jsonb is struct
```

Comment:
Why the `Bigquery` cases is removed?

Reply:
Because I found that our code went through some refactoring to include the data types, bigquery is no longer a special option and can be used with other sinks using common creation logic